code/__HELPERS/icons.dm

```diff
@@ -1006,4 +1006,22 @@ var/global/list/humanoid_icon_cache = list()
 //Images have dir without being an atom, so they get their own definition.
 //Lame.
 /image/proc/setDir(newdir)
-	dir = newdir
+	dir = newdir
+
+// Used to make the frozen item visuals for Freon.
+
+/proc/make_frozen_visual(var/obj/F)

```

Comment:
/obj/proc/make_frozen_visual()

Also there's probably a better file to put this in.